WalletWasabi.Fluent/ViewModels/Wallets/Send/ChangeAvoidanceSuggestionViewModel.cs

```diff
@@ -66,39 +66,36 @@ public partial class ChangeAvoidanceSuggestionViewModel : SuggestionViewModel
 
 		await foreach (IEnumerable<SmartCoin> selection in selectionsTask.ConfigureAwait(false))
 		{
-			if (selection.Any())
```

Comment:
> This can be a separate PR.

It can be, but I think it's clear what changed. 
You made sure we don't return an empty-set of "coins" and you removed its nullability check.